examples/rails-secrets/cookbooks/custom-rails-secrets/README.md

```diff
@@ -0,0 +1,39 @@
+# rails-secrets
+
+This recipe uploads a `secrets.yml` file to `/data/application_name/shared/config/secrets.yml`. 
+
+## Installation
+
+For simplicity, we recommend that you create the cookbooks directory at the root of your application. If you prefer to keep the infrastructure code separate from application code, you can create a new repository.
+
+Our main recipes have the `rails-secrets` recipe but it is not included by default. To use the `rails-secrets` recipe, you should copy this recipe `custom-rails-secrets`. You should not copy the actual `rails-secrets` recipe. That is managed by Engine Yard.
+
+1. Edit `cookbooks/ey-custom/recipes/after-main.rb` and add
+
+      ```
+      include_recipe 'custom-rails-secrets'
+      ```
+
+2. Edit `cookbooks/ey-custom/metadata.rb` and add
+
+      ```
+      depends 'custom-rails-secrets'
+      ```
+
+3. Copy `examples/rails-secrets/cookbooks/custom-rails-secrets` to `cookbooks/`
+
+      ```
+      cd ~ # Change this to your preferred directory. Anywhere but inside the application
+
+      git clone https://github.com/engineyard/ey-cookbooks-stable-v5
+      cd ey-cookbooks-stable-v5
+      cp examples/rails-secrets/cookbooks/rails-secrets /path/to/app/cookbooks/
+      ```
+
+If you do not have `cookbooks/ey-custom` on your app repository, you can copy `examples/rails-secrets/cookbooks/ey-custom` to `/path/to/app/cookbooks`.
+
+## Customizations
+
+The recipe supports multi-application environments. For each application, you need to copy the application's `secrets.yml` to `/path/to/app/cookbooks/rails-secrets/templates/default/secrets.yml.#{app_name}.erb`
```

Comment:
This is mandatory so we should add it to the Installation instructions too. Do we need to add cookbooks/rails-secrets aside from cookbooks/custom-rails-secrets?

Can we try to avoid that by using the templates from custom-rails-secrets? That will probably make rails-secrets useless though.